### ISSUE TITLE
Displaying images in Google Colab

### ISSUE THREAD

**Issue by Kipkurui-mutai** [python-pillow/Pillow#5286]
### What did you do?
`Image.show()`

### What did you expect to happen?
Display the Image

### What actually happened?
<PIL.PngImagePlugin.PngImageFile image mode=RGB size=450x253 at 0x7FFB7B401C50>

### What are your OS, Python and Pillow versions?

* OS: Windows
* Python: 3.9
* Pillow: 7.0.0

```python
display(image)
```

**Comment by radarhere** [comment]
At the moment, this is somewhat confusing.

For one thing, you can't simply call `Image.show()` - unless `Image` is the name of your variable.
You could
```python
from PIL import Image
im = Image.new("RGB", (100, 100))
im.show()
```

For another, where does the `display()` method come from? That's not a part of Pillow.

Would you be able to provide a more complete example?

**Comment by Kipkurui-mutai** [comment]
![Capture](https://user-images.githubusercontent.com/62174033/109224355-e3929400-77cc-11eb-946c-3bc765c2971d.PNG)
So I tried `im.show()` and couldn't print anything. I tried `display(im)` and it worked for me.

**Comment by Kipkurui-mutai** [comment]
@radarhere I am using Colab.

**Comment by Kipkurui-mutai** [comment]
@radarhere I have tried `im.show()` using my terminal and it works fine. I do not know why it's not working on https://colab.research.google.com

**Comment by nulano** [comment]
From https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.show:

> Displays this image. This method is mainly intended for debugging purposes.
>
> This method calls PIL.ImageShow.show() internally. You can use PIL.ImageShow.register() to override its default behaviour.
>
> The image is first saved to a temporary file. By default, it will be in PNG format.
>
> On Unix, the image is then opened using the display, eog or xv utility, depending on which one can be found.
>
> On macOS, the image is opened with the native Preview application.
>
> On Windows, the image is opened with the standard PNG display utility.

The display, eog and xv utilities require a display; Google Colab doesn't give you access to the display, only a terminal.

**Comment by Kipkurui-mutai** [comment]
@nulano  Other libraries like those for visualization are displaying there images on Colab well. Does this mean it cannot be done with Pillow?

**Comment by radarhere** [comment]
I've created PR #5289 to suggest adding this functionality to Pillow. Even if it is merged though, it might take a while to get bundled into Google Colab, since the current version in Colab is Pillow 7.0, a bit behind the current version of 8.1.0.

**Comment by Kipkurui-mutai** [comment]
@radarhere will this make me a contributor or I must write code?

**Comment by radarhere** [comment]
No, 'Contributors' isn't a manual list that we add you to, it's an automatic one - you must write a commit, and have that commit be merged in.

**Comment by Kipkurui-mutai** [comment]
@radarhere so how can I contribute ? I see almost everything is pretty much done.

**Comment by radarhere** [comment]
If you're asking what the next step is to the change being accepted, this is the review stage. So anyone, yourself included, can take a look at it and make comments for how it could be improved, or create a pull request in my Pillow fork with commits that they would like to be added to it.

After that, one of the Pillow members needs to decide to merge it, and then it will be part of the next Pillow release. So the soonest it could be released is April 1.

**Comment by Kipkurui-mutai** [comment]
@radarhere I really wanted to contribute to that functionality. Guess I should probably look out for other improvements and Create a PR myself.

**Comment by hugovk** [comment]
A good place to start is with documentation. 

You could add https://github.com/python-pillow/Pillow/pull/5289 to the release notes, perhaps in the API Additions section? 

https://github.com/python-pillow/Pillow/blob/master/docs/releasenotes/8.2.0.rst

And does it also belong elsewhere in docs?

**Comment by Kipkurui-mutai** [comment]
@hugovk  Where exactly, I am new in contributing to open source, 49/50 checks has already been passed. Does this mean #5289 has already been updated in the release notes?

**Comment by hugovk** [comment]
No, the checks are unrelated. That PR doesn't have release notes. If you'd like to contribute them, you can edit https://github.com/python-pillow/Pillow/blob/master/docs/releasenotes/8.2.0.rst and create a new pull request.

**Comment by radarhere** [comment]
It's also apparent that https://pillow--5289.org.readthedocs.build/en/5289/reference/ImageShow.html doesn't mention IPythonViewer at all. If you'd like to fix that, you can modify https://github.com/python-pillow/Pillow/blob/master/docs/reference/ImageShow.rst.

Once your change is ready, you can go to https://github.com/radarhere/Pillow/compare/ipythonviewer...Kipkurui-mutai:master to create a pull request for #5289 specifically.
